includes/modules/payment/paypal/tpl_ec_button.php

```diff
@@ -7,7 +7,9 @@
  * @version $Id: Scott C Wilson 2022 Feb 02 Modified in v1.5.8-alpha $
  */
 
-$paypalec_enabled = (defined('MODULE_PAYMENT_PAYPALWPP_STATUS') && MODULE_PAYMENT_PAYPALWPP_STATUS == 'True');
+// PayPal module cannot be used for purchase > $10,000 USD equiv
+require_once DIR_FS_CATALOG . DIR_WS_MODULES . 'payment/paypal/paypal_currency_check.php';
+$paypalec_enabled = (defined('MODULE_PAYMENT_PAYPALWPP_STATUS') && MODULE_PAYMENT_PAYPALWPP_STATUS == 'True'  && paypalUSDCheck($_SESSION['cart']->total));
```

Comment:
Same comment as the login page's header_php.php.

Reply:
Yes agreed. It is clearer and easier to see.